### PR TITLE
[Lorem Ipsum] Add options to Paste to active app and then copy to clipboard

### PR DESCRIPTION
**Pull request by jfkisafk**
## Description

Adds feature to paste to active app and then copy to clipboard.

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

* closes https://github.com/raycast/extensions/issues/13945

### CODE REVIEW

File: extensions/lorem-ipsum/package.json [hunk 1]
```diff
@@ -102,8 +102,13 @@
         {
           "value": "paste",
           "title": "Paste to Active App"
+        },
+        {
+          "value": "pasteAndCopy",
+          "title": "Paste and Copy to Clipboard"
         }
       ],
+      "default": "clipboard",
```
Comment (AntonNiklasson) [inline extensions/lorem-ipsum/package.json:111]: I guess this doesn't alter the previous behaviour?

**Review by AntonNiklasson — APPROVED** [review]
Looks great, thanks for taking the time to implement this! 🥇

**Comment by jfkisafk** [comment]
Hello @pernielsentikaer , can you please help review/merge this?

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**